src/task/Task.ts

```diff
@@ -18,14 +17,13 @@ export class Task extends BaseEntity {
     @Column()
     public status!: "complete" | "open" | "pending";
 
-    @ManyToOne(() => User, user => user.tasks) 
-    public author!: User;
-    
-    @Column("uuid") 
-    public authorId!: User["id"];
 
-    // @ManyToOne(() => Board, board => board.tasks) board!: Board;
-    // @Column() boardId!: Board["id"];
+    @ManyToOne(() => User, user => user.tasks) author!: User;
+	@Column() authorId!: User["id"];
+
+    @ManyToOne(() => Board, board => board.tasks) board!: Board;
+    @Column() boardId!: Board["id"];
```

Comment:
`authorId` & `boardId` need to have `@Column("uuid")` in order for TypeORM to query them properly.